src/Http/HttpAbstractions.slnf

```diff
@@ -40,18 +42,24 @@
       "src\\Http\\WebUtilities\\perf\\Microbenchmarks\\Microsoft.AspNetCore.WebUtilities.Microbenchmarks.csproj",
       "src\\Http\\WebUtilities\\src\\Microsoft.AspNetCore.WebUtilities.csproj",
       "src\\Http\\WebUtilities\\test\\Microsoft.AspNetCore.WebUtilities.Tests.csproj",
-      "src\\Http\\samples\\MinimalSample\\MinimalSample.csproj",
```

Comment:
Can we leave this in the slnf?

Reply:
Brought this back. Surprisingly, the build in VS works now so I assume it was including the other properties that was required.